continuous-integration/deployment/check_expected_artifacts.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+
+def _expected_artifacts():
+    parent_dir = Path(__file__).resolve().parent
+    artifacts_file = parent_dir / "expected-artifacts.txt"
+    artifacts = artifacts_file.read_text().splitlines()
+    artifacts.sort()
+    version_file = parent_dir / "expected-version.txt"
+    version = version_file.read_text().strip()
+    return [artifact.format(version=version) for artifact in artifacts]
+
+
+def _present_artifacts():
+    artifacts_dir = Path(__file__).resolve().parents[2] / "dist"
```

Comment:
```suggestion
    artifacts_dir = Path(__file__).resolve().parents[2] / "distribution"
```
The reason for this is that it makes the workflow cleaner, the path itself becomes `distribution`

Reply:
Sure, done.